pyccel/parser/syntax/headers.py

```diff
@@ -385,7 +384,7 @@ def expr(self):
         arg_ = self.arg
         if isinstance(arg_, MacroList):
             return tuple(arg_.expr)
-        arg = Symbol(str(arg_))
+        arg = PyccelSymbol(str(arg_))
```

Comment:
Given that `PyccelSymbol` subclasses `str`, do we really need to convert its argument to a string?

Reply:
25403cf